test/interactive_tarantool.lua

```diff
@@ -263,18 +288,29 @@ function M.new(opts)
     local res = setmetatable({
         ph = ph,
         _readahead_buffer = '',
-        _prompt = 'tarantool> ',
+        _prompt = opts.prompt or 'tarantool> ',
     }, mt)
 
     -- Log child's stderr.
     res:_start_stderr_logger()
 
+    return res
+end
+
+function M.new_debugger(opts)
+    opts.prompt = opts.prompt or 'luadebug> '
```

Comment:
Nit: I would add `local opts = opts or {}` like in `_new_internal()` to allow to call this function without arguments (if no options are needed).

Reply:
Done.